tests/strings/vararg.cpp

```diff
@@ -200,18 +200,23 @@ void VarArgTestCase::Sscanf()
 {
     int i = 0;
     char str[20];
-    wchar_t wstr[20];
 
     wxString input("42 test");
 
     wxSscanf(input, "%d %s", &i, &str);
     CPPUNIT_ASSERT( i == 42 );
     CPPUNIT_ASSERT( wxString(str) == "test" );
 
+#ifndef __MINGW32__
+    // disable this test on mingw to prevent segmentation fault:
```

Comment:
Should this be always done or just for some value of `__USE_MINGW_ANSI_STDIO`?

Reply:
It seems only with `__USE_MINGW_ANSI_STDIO=1`. For example, the code below works. But when enabling `#include <string>` it crashes.

```c++
//#include <string>
#include <wchar.h>

#define VALUE_TO_STRING(x) #x
#define VALUE(x) VALUE_TO_STRING(x)
#define VAR_NAME_VALUE(var) #var "="  VALUE(var)

int main()
{
#pragma message(VAR_NAME_VALUE(__USE_MINGW_ANSI_STDIO))

  wchar_t sentence [] = L"42 test";
  wchar_t str [20];
  int i;

  swscanf (sentence,L"%d %s",&i,str);
  wprintf (L"%s -> %d\n",str,i);

  return 0;
}

```

Reply:
Unfortunately something is pulling in stl headers, probably Catch2.